server/bootstrap/routes.js

```diff
@@ -5,16 +5,18 @@ var IndexController = require(process.env.ROOT + '/server/mvc/controllers/IndexC
 var AdminController = require(process.env.ROOT + '/server/mvc/controllers/AdminController.js');
 var Admin_UsersController = require(process.env.ROOT + '/server/mvc/controllers/Admin/UsersController.js');
 var ProfileController = require(process.env.ROOT + '/server/mvc/controllers/ProfileController.js');
+var BattlenetController = require(process.env.ROOT + '/server/mvc/controllers/BattlenetController.js');
 
 module.exports = function (app, logs, passport) {
   app.get('/auth/bnet', passport.authenticate('bnet'));
   app.get('/auth/bnet/callback', passport.authenticate('bnet', {failureRedirect: '/'}), function(req, res){
-    res.redirect('/');
+    res.redirect('/bnet');
```

Comment:
That's true. We could do that.
Fetching profile data from Bnet is a bit slow, though. And so, to improve UX, I'd recommend we show a page with a spinner/loading GIF and then create the user via XHR.